services/src/main/java/org/fao/geonet/api/standards/StandardsApi.java

```diff
@@ -394,4 +412,22 @@ public String getEditorAssociatedPanelConfiguration(
             name, schemasProcessed.toString()));
 
     }
+
+    /**
+     * Checks if the user profile is allowed to batch edit metadata.
+     *
+     * @param userSession
+     */
+    private void checkUserProfileToBatchEditMetadata(UserSession userSession) {
```

Comment:
This code is repeated in other places. Also there is similar code for other features (`checkUserProfileToDeletePublishedMetadata`, `checkUserProfileToImportMetadata`). 

Could be good to refactor all these methods in a utility class in the `services` module: `UserProfileUtil` or similar with these methods.

Reply:
For now in this pull request can be refactor only `checkUserProfileToBatchEditMetadata`, that should be fine also.

Reply:
i am good with refactoring to a common place. I will make the code change. We should make a common name for it (i.e checkUserProfileLevel)

Reply:
@josegar74 

I did a total refactoring including checkUserProfileToDeletePublishedMetadata and checkUserProfileToImportMetadata to the more general util method. Please take a look